src/Tabs.tsx

```diff
@@ -1 +1,39 @@
-// implement a component here
+import React from 'react';
+import { Tab } from './react-app-env';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const findTabById = (tabId: string): Tab | undefined => {
+    return tabs.find(tab => tab.id === tabId);
+  };
+
+  const currentTab = findTabById(selectedTabId) || tabs[0];
+
+  return (
+    <>
+      <div className="tabs">
+        {tabs.map(tab => (
+          <button
+            key={tab.id}
+            type="button"
+            onClick={() => onTabSelected(tab)}
```

Comment:
Will be better to change selected tab only if selected tab id is not equal to the current tab id